### PR TITLE
Remove last direct mentions of GL1

### PR DESCRIPTION
**Pull request by mvaligursky**
- small cleanup

### CODE REVIEW

**Comment by LeXXik** [comment]
Probably here too:

https://github.com/playcanvas/engine/blob/56d4dedd85b6c2aea5ff664b65292f94be18451e/README.md?plain=1#L43

File: README.md [hunk 1]
```diff
@@ -40,7 +40,7 @@ PlayCanvas is used by leading companies in video games, advertising and visualiz
 
 PlayCanvas is a fully-featured game engine.
 
-* 🧊 **Graphics** - Advanced 2D + 3D graphics engine built on WebGL 1 & 2.
+* 🧊 **Graphics** - Advanced 2D + 3D graphics engine built on WebGL 2 & WebGPU.
```
Comment (slimbuck) [inline README.md:43]:
```suggestion
* 🧊 **Graphics** - Advanced 2D + 3D graphics engine built on WebGL2 & WebGPU.
```

File: src/platform/graphics/webgl/webgl-graphics-device.js [hunk 2]
```diff
@@ -930,7 +930,7 @@ class WebglGraphicsDevice extends GraphicsDevice {
         this.extCompressedTextureASTC = this.getExtension('WEBGL_compressed_texture_astc');
         this.extParallelShaderCompile = this.getExtension('KHR_parallel_shader_compile');
 
-        // iOS exposes this for half precision render targets on both Webgl1 and 2 from iOS v 14.5beta
+        // iOS exposes this for half precision render targets on Webgl 2 from iOS v 14.5beta
```
Comment (slimbuck) [inline src/platform/graphics/webgl/webgl-graphics-device.js:933]:
```suggestion
        // iOS exposes this for half precision render targets on WebGL2 from iOS v 14.5beta
```

**Review by slimbuck — APPROVED** [review]
Approve with suggestions

**Pull request merged**